config/302-pac-configmap.yaml

```diff
@@ -42,8 +42,9 @@ data:
   # Add extra IPS (ie: 127.0.0.1) or networks (127.0.0.0/16) separated by commas.
   bitbucket-cloud-additional-source-ip: ""
 
-  # The default time to wait for a pipelineRun
-  default-pipelinerun-timeout: "2h"
+  # Global setting for pipelinerun timeout regardless of how the user
```

Comment:
so there won't be a default timeout? configurable across all

Reply:
```
 timeout will be deprecated in future versions. Consider using timeouts instead.
```
interesting 🤔 we need to keep a watch when ..
or we need to start handling both